CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Fixes
```

Comment:
Bump the version to `1.5.0` and change the `1.4.0rcX` to `1.4.0` :wink:

Reply:
Done!

Reply:
`[UNRELEASED] 1.5.0` :wink: